testing/sdk_cmd.py

```diff
@@ -154,6 +147,27 @@ def run_raw_cli(cmd, print_output=True):
     return result.returncode, stdout, stderr
 
 
+
+def run_raw_cli(cmd, print_output=True):
+    """Runs the command with `dcos` as the prefix to the shell command
+    and returns the resulting output (stdout separated from stderr by a newline).
```

Comment:
This docstring is incorrect as this returns an `rc, stdout, stderr` tuple.

Reply:
Fixed the stale doc string